sdks/python/apache_beam/transforms/display_test.py

```diff
@@ -22,8 +22,16 @@
 from datetime import datetime
 import unittest
 
+import hamcrest as hc
+
 import apache_beam as beam
-from apache_beam.transforms.display import HasDisplayData, DisplayData, DisplayDataItem
+from apache_beam.transforms.display import *
+
+
+def make_nspace_display_data(component):
```

Comment:
It looks like only a couple tests classes are using this-- should they all?

Note that the transforms and tests we have in the core SDK are likely to get copy-pasted many times over their life, for future SDK transforms and by users. Your implementation here establishes the recipe for how to implement and test display data.